pkgs/top-level/ocaml-packages.nix

```diff
@@ -1101,6 +1101,8 @@ let
       then janeStreet.ppx_variants_conv
       else callPackage ../development/ocaml-modules/janestreet/ppx-variants-conv.nix {};
 
+    ppx_yojson_conv_lib = callPackage ../development/ocaml-modules/janestreet/ppx-yojson-conv-lib.nix {};
```

Comment:
Why is this file there, rather than at `ocaml-modules/ppx_yojson_conv_lib/default.nix`?

Reply:
I thought that packages from janestreet should be under `ocaml-modules/janestreet/`. 
I've moved it to `ocaml-modules/ppx_yojson_conv_lib/`